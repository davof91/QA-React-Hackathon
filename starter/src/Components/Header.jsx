import React from 'react';
import logo from './images/qalogo.svg';

const Header = () => {
    return (
        <>
            <header>
                <nav class="navbar navbar-expand-sm">
                    <a class="navbar-brand" href="https://qa.com" target="_blank" rel="noreferrer">
                    <img alt="QA Ltd" src={logo} width="100" />
                    </a>
                    <a href="/" class="navbar-brand">Todo App</a>
                </nav>
            </header>
        </>
    );
}

export default Header;