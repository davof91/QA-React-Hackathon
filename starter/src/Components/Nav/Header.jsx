import React from 'react';

const Header = () => {
    return (
        <header className="container sticky-top" id="qa-navbar">
            <nav className="navbar navbar-expand-md navbar-light">
            <a className="navbar-brand" href="https://qa.com"><img src="../images/qa.jpeg" alt="logo" /></a>
            <button className="navbar-toggler" type="button" data-toggle="collapse" data-target="#navBarNav" aria-controls="navBarNav" aria-expanded="false" aria-label="Toggle navigation">
                <span className="navbar-toggler-icon"></span>
            </button>
            <div className="collapse navbar-collapse" id="navbarNav">
                <ul className="navbar-nav">
                    <li className="nav-item"><a href="/" class="nav-link">Home</a></li>
                    <li className="nav-item"><a href="/schedule" class="nav-link">Schedule</a></li>
                    <li className="nav-item"><a href="/signup" class="nav-link">Sign-up</a></li>
                </ul>
            </div>
        </nav>
        </header>
    );
}

export default Header;