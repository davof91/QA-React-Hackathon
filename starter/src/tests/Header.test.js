import React from 'react';
import { create } from 'react-test-renderer';
import Header from '../Components/Header';

test('Header matches snapshot', () => {
    const header = create(<Header />);
    expect(header.toJSON()).toMatchSnapshot();
  });