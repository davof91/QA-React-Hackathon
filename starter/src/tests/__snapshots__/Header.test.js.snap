// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<header>
  <nav
    class="navbar navbar-expand-sm"
  >
    <a
      class="navbar-brand"
      href="https://qa.com"
      rel="noreferrer"
      target="_blank"
    >
      <img
        alt="QA Ltd"
        src="qalogo.svg"
        width="100"
      />
    </a>
    <a
      class="navbar-brand"
      href="/"
    >
      Todo App
    </a>
  </nav>
</header>
`;
